packages/common/src/ezbackend.ts

```diff
@@ -171,7 +171,6 @@ export class EzBackend extends EzApp {
 
         this.setInit('Manage Event Subscriptions', async (instance, opts) => {
             instance.subscribers = []
-            instance.subscribers.push(createModelSubscriber(instance))
```

Comment:
Is this intentional?

Reply:
Yeap, its intentional, previous I was using the TypeORM hooks (AKA subscribers) to listen for events, but I think that will cause confusing behaviour for people when they create their own endpoints, especially because the typeorm hooks are quite unreliable

It has been moved to api-generator instead